training/distributed_training/tensorflow/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Where should the data be downloaded from? How do you prepare it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>For #2 of Important Caveats, use this link for "this reference" instead: https://docs.aws.amazon.com/sagemaker/latest/dg/distributed-troubleshooting-data-parallel.html#distributed-ts-data-parallel-efa-sg</p>